tck/src/main/java/org/eclipse/microprofile/rest/client/tck/interfaces/InvocationCallbackClientAsync.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.eclipse.microprofile.rest.client.tck.interfaces;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.client.InvocationCallback;
+
+@Path("/string")
+public interface InvocationCallbackClientAsync {
```

Comment:
Isn't this missing `@Dependent` and `@RegisterRestClient`?

Reply:
No, those annotations are only required when using CDI.  For these test cases, I'm using the `RestClientBuilder` - if we wanted to use this same client interface in a test that involved CDI, then we would definitely need to add `@RegisterRestClient` - as for `@Dependent`, I'm hoping that that won't be necessary (even in the CDI case) after PR #83 is merged.

This answer should also apply to the questions in `SimpleGetApiAsync.java` (which is used in a CDI case) and `StringResponseClientAsync.java`.